cli/cliui/agent.go

```diff
@@ -414,37 +420,45 @@ func (d ConnDiags) splitDiagnostics() (general, client, agent []string) {
 	}
 
 	if d.ConnInfo.DisableDirectConnections {
-		general = append(general, "❗ Your Coder administrator has blocked direct connections")
+		general = append(general, "❗ Your Coder administrator has blocked direct connections\n"+
+			"   https://coder.com/docs/networking/troubleshooting#disabled-deployment-wide")
 		if !d.Verbose {
 			return general, client, agent
 		}
 	}
 
 	if !d.ConnInfo.DERPMap.HasSTUN() {
-		general = append(general, "The DERP map is not configured to use STUN")
+		general = append(general, "❗ The DERP map is not configured to use STUN\n"+
+			"   https://coder.com/docs/networking/troubleshooting#no-stun-servers")
 	} else if d.LocalNetInfo != nil && !d.LocalNetInfo.UDP {
-		client = append(client, "Client could not connect to STUN over UDP")
+		client = append(client, "Client could not connect to STUN over UDP\n"+
+			"   https://coder.com/docs/networking/troubleshooting#udp-blocked")
 	}
 
 	if d.LocalNetInfo != nil && d.LocalNetInfo.MappingVariesByDestIP.EqualBool(true) {
-		client = append(client, "Client is potentially behind a hard NAT, as multiple endpoints were retrieved from different STUN servers")
+		client = append(client, "Client is potentially behind a hard NAT, as multiple endpoints were retrieved from different STUN servers\n"+
+			"   https://coder.com/docs/networking/troubleshooting#Endpoint-Dependent-Nat-Hard-NAT")
 	}
 
 	if d.AgentNetcheck != nil && d.AgentNetcheck.NetInfo != nil {
 		if d.AgentNetcheck.NetInfo.MappingVariesByDestIP.EqualBool(true) {
-			agent = append(agent, "Agent is potentially behind a hard NAT, as multiple endpoints were retrieved from different STUN servers")
+			agent = append(agent, "Agent is potentially behind a hard NAT, as multiple endpoints were retrieved from different STUN servers\n"+
+				"   https://coder.com/docs/networking/troubleshooting#Endpoint-Dependent-Nat-Hard-NAT")
 		}
 		if !d.AgentNetcheck.NetInfo.UDP {
-			agent = append(agent, "Agent could not connect to STUN over UDP")
+			agent = append(agent, "Agent could not connect to STUN over UDP\n"+
+				"   https://coder.com/docs/networking/troubleshooting#udp-blocked")
 		}
 	}
 
 	if d.ClientIPIsAWS {
-		client = append(client, "Client IP address is within an AWS range (AWS uses hard NAT)")
+		client = append(client, "Client IP address is within an AWS range (AWS uses hard NAT)\n"+
+			"   https://coder.com/docs/networking/troubleshooting#Endpoint-Dependent-Nat-Hard-NAT")
 	}
 
 	if d.AgentIPIsAWS {
-		agent = append(agent, "Agent IP address is within an AWS range (AWS uses hard NAT)")
+		agent = append(agent, "Agent IP address is within an AWS range (AWS uses hard NAT)\n"+
+			"   https://coder.com/docs/networking/troubleshooting#Endpoint-Dependent-Nat-Hard-NAT")
```

Comment:
I believe it's possible for mapvarydest to be false for an agent behind AWS' NAT, so I've included this link on both diagnostics.